### ISSUE TITLE
Services should not allow jobs to start before upload is complete

### ISSUE THREAD

**Issue by levinas** [PATRIC3/patric3_website#543]
In the current services, say assembly, the user is allowed to click 'Assemble' while upload is still in progress. This results in a quick failure as the data is not available at the time of the launch. Should this be handled by the front end of backend?

**Comment by olsonanl** [comment]
Hm, good catch. I’m thinking the easiest proper behavior is for the submit button to be disabled until the upload finishes.

More sophisticated would allow the submit to happen but wait until the upload finishes, but since the upload is handled by the page code that’s not entirely feasible.

For large uploads I think we need a nicely packaged native app.

—bob

On Nov 22, 2015, at 9:28 PM, Fangfang Xia <notifications@github.com<mailto:notifications@github.com>> wrote:


In the current services, say assembly, the user is allowed to click 'Assemble' while upload is still in progress. This results in a quick failure as the data is not available at the time of the launch. Should this be handled by the front end of backend?

**Comment by rkenyon** [comment]
Not sure to whom this needs to be assigned.

**Comment by olsonanl** [comment]
This needs to go to the webside folks to add an interlock between the file upload and the enabling of the submit button.

**Comment by dawenx** [comment]
this is an ancient issue, but it's the same as #1784 and #1383. should be resolved soon.

**Comment by mshukla1** [comment]
@dawenx I like to review this fix during the  meeting Today, or with you separately. 

I tried it on alpha using the assembly service. I tried to upload two files directly from the job submission form. As soon as I selected he file and it started uploading, the file selection text box was grayed out without showing file name. When I clicked on the side arrow to add pair of files to selection, it generated red flag. I found this very confusing. 

Screenshot below. 

<img width="386" alt="screen shot 2017-12-12 at 5 44 12 am" src="https://user-images.githubusercontent.com/1236463/33883113-2e9d26d6-df00-11e7-8b53-062d08ebf77c.png">
